### ISSUE TITLE
`ion-select-popover` does not render with Stencil

### ISSUE THREAD

**Issue by tricki** [ionic-team/ionic#17514]
# Bug Report

**Ionic version:**
[x] **4.x**

**Current behavior:**
`ion-select` with `interface="popover"` does not display anything in the popover when using Ionic with Stencil because `ion-select-popover` is not rendered.

**Expected behavior:**
Popover renders correctly.

**Steps to reproduce:**
Clone the PWA Toolkit and add a `ion-select` with `interface="popover"`.

**Related code:**

```
<ion-select interface="popover">
  <ion-select-option>Item 1</ion-select-option>
  <ion-select-option>Item 2</ion-select-option>
</ion-select>
```

**Other information:**
Workaround: https://github.com/ionic-team/stencil/issues/775

**Ionic info:**
```
Ionic:

   ionic (Ionic CLI) : 4.10.3

System:

   NodeJS : v10.15.0
   npm    : 6.4.1
   OS     : Windows 10
```

**Comment by djoks** [comment]
I can confirm this bug still exists.

**Comment by santoshyadav198613** [comment]
Hi @brandyscarney ,

Let me check this, if not done.

**Comment by simonhaenisch** [comment]
https://github.com/ionic-team/stencil/issues/775

Workaround: add a `<ion-select-popover style={{ display: 'none' }} />` to your DOM.

Reason is that the `ion-select-popover` element doesn't get included in the component's bundle unless it's part of the template.

But would be nice to have a proper fix.

**Comment by simonhaenisch** [comment]
Can confirm this is fixed with `@stencil/core@1.0.x` and `@ionic/core@4.6.0-dev.201906131724.645b9a9` (:

**Comment by brandyscarney** [comment]
Thank you @simonhaenisch! Please update to core version `4.6.2`. If you are still seeing this issue, please create a new issue. 🙂
